04_pig_latin/pig_latin.rb

```diff
@@ -0,0 +1,28 @@
+def foo(value, number)
+  "#{value[number..-1]}#{value[0..number-1]}"
+end
+def pig_latin_for_one_word(string1)
+  vowel = ["a", "e", "i", "o", "u"]
+  i = 0
+  loop do
```

Comment:
:+1:

Reply:
Let's try to avoid `loop`, `for`, `while`, and so on from now on. Rather go for `each`, `map` or other `Enumerable`-based iterators.